cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/ClusteredAgentScheduler.java

```diff
@@ -87,7 +83,8 @@ private void runAgents() {
         activeAgents.putAll(thisRun);
         for (final Map.Entry<String, Long> toRun : thisRun.entrySet()) {
             final AgentExecutionAction exec = agents.get(toRun.getKey());
-            agentExecutionPool.submit(new AgentJob(toRun.getValue(), exec, this));
+            Future s = agentExecutionPool.submit(new AgentJob(toRun.getValue(), exec, this));
+            assert !s.isCancelled();
```

Comment:
FindBugs was complaining that I was ignoring the result of these futures. (not sure why it was failing here, since this was otherwise untouched). Is there a neater way to do this? @cfieber